### DISCUSSION TITLE
the api used by aws-cli

### DISCUSSION THREAD

**Discussion by YahavFes** [aws/aws-cli#7021] [Q&A] [answered]
I would like to know in what API does aws-cli use.
I found an example of old version api [here](https://github.com/aws/aws-sdk-js/blob/master/apis/iam-2010-05-08.min.json), but I want to know where can I find the most updated API :)

Thanks in advance.

**Comment by nateprewitt** [comment, marked as answer]
Hi @YahavFes, the CLI is built on [Botocore](https://github.com/boto/botocore) which contains all of the models used for AWS services. You can find them [here](https://github.com/boto/botocore/tree/develop/botocore/data).
